README.md

```diff
@@ -30,12 +30,37 @@ jobs:
     - uses: actions/checkout@v2
     - name: Setup tmate session
       uses: mxschmitt/action-tmate@v3
+      with:
```

Comment:
Could we add a comment here saying that this requires a public SSH key to be added to the GitHub profile, and a direct link to the profile where that could be done for convenience?

Reply:
Can you make a proposal?

Reply:
To this link I guess? https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account

Reply:
I was thinking more about https://github.com/settings/ssh/new, but both links would work for me.

Reply:
In addition to linking to https://github.com/settings/ssh/new, I would like to see in a comment that the key is needed to protect any secrets that are available in the workflow step, and that the `limit-access-to-actor` setting can be set to `false` for convenience if there is no risk for spilling secrets.

In my mind, this improvement is needed for this PR to be merged.